playback/src/player.rs

```diff
@@ -1043,11 +1091,11 @@ impl Future for PlayerInternal {
 
 impl PlayerInternal {
     fn position_pcm_to_ms(position_pcm: u64) -> u32 {
-        (position_pcm * 10 / 441) as u32
+        (position_pcm as f64 * MS_PER_SAMPLE).round() as u32
     }
 
     fn position_ms_to_pcm(position_ms: u32) -> u64 {
-        position_ms as u64 * 441 / 10
+        (position_ms as f64 * SAMPLES_PER_MS).round() as u64
```

Comment:
Thinking along with me: could these now inadvertently return a position that doesn't exist, because of rounding up? i.e. should you ensure the upper bounds are within limits?

Reply:
This I do not know? Gstreamer for example interprets a seek beyond the length of a track as skipping to the beginning next track if there is one or it just signals an EOS (end of stream). I'm not sure if this is a **standard** way to handle that situation, as in that's how individual decoders would handle it or if it's just a Gstreamer thing. If there's any doubt we could always just skip the rounding and it would just truncate? We would never be more than 1 ms or 1 page off. I can't imagine that making much of any difference.

Reply:
Done.